### PR TITLE
[ DONATOR ITEM ] Purple Outfit Set

### PR DESCRIPTION
**Pull request by Crumpaloo**
## About The Pull Request

A Donator item requested by Thedragmeme, PR contains 3 new objects and their directionals/item states meant to act like one outfit broken up into several sprites:

- Caged Purple Dress
- Short Purple Coat
- Flower Neck Collar

image

## How This Contributes To The Skyrat Roleplay Experience

Allows Thedragmeme to use their donator item slot.

### CODE REVIEW

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 1]
```diff
@@ -562,3 +562,18 @@
 	name = "Bumbling Wolfgirl Plushie"
 	item_path = /obj/item/toy/plush/skyrat/plushie_elofy
 	ckeywhitelist = list("october23")
+
+/datum/loadout_item/under/jumpsuit/caged_dress
+	name = "Caged Purple Dress"
+	item_path = /obj/item/clothing/under/caged_dress
+	ckeywhitelist = list("Thedragmeme")
```
Comment (Zonespace27) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]: no capitalization

Comment (Crumpaloo) [reply modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]: Took them out, should be good now.

**Review by Ebin-Halcyon — CHANGES_REQUESTED** [review]
Needs cut outs of the arms on the side state. Those don't overlay on clothing, only hands don't need a hole cut out for it.

**Comment by Crumpaloo** [comment]
> Needs cut outs of the arms on the side state. Those don't overlay on clothing, only hands don't need a hole cut out for it.
Updated them so the side directionals have the cut-outs.
image

**Pull request closed without merging**